docker/Dockerfile.template

```diff
@@ -0,0 +1,26 @@
+FROM node:7.9.0
```

Comment:
Guessing this is about ~500-600MB. Would prefer to see a Node microcontainer. Would likely bring the container size down about half a gig. https://www.iron.io/microcontainers-tiny-portable-containers/